src/services/CertificateDownloadService.ts

```diff
@@ -0,0 +1,50 @@
+import {Configuration} from "../utils/Configuration";
+import {S3BucketService} from "./S3BucketService";
+import {Service} from "../models/injector/ServiceDecorator";
+import S3 from "aws-sdk/clients/s3";
+import {S3BucketMockService} from "../../tests/models/S3BucketMockService";
```

Comment:
why do we have this line here? Pls make sure we dont have anything additional in file than what is required.

Reply:
deleted